src/pydata_sphinx_theme/assets/styles/variables/_color.scss

```diff
@@ -189,6 +189,11 @@ $pst-semantic-colors: (
     "light": rgba(23, 23, 26, 0.2),
     "dark": #{map-deep-get($color-palette, "gray", "700")},
   ),
+  "blockquote-notch": (
+    // gray-500 has a contrast ratio > 3.0 for foundation white and black
```

Comment:
IIRC the dark mode foundation isn't pure black, did you check against the color we actually use in dark mode?

Reply:
foundation white and foundation black ;)

Reply:
But I'm going to tweak this again

Reply:
Because I want it to contrast a bit more with the dark surface color (the blockquote background)